platform/search/bundle.js

```diff
@@ -109,7 +109,8 @@ define([
                             "topic",
                             "GENERIC_SEARCH_ROOTS",
                             "USE_LEGACY_INDEXER",
-                            "openmct"
+                            "openmct",
+                            "objectService"
```

Comment:
lets add this in place of modelService, since we are replacing it

Reply:
updated!